rlclientlib/serialization/payload_serializer.h

```diff
@@ -69,7 +69,7 @@ namespace reinforcement_learning {
 
     struct multi_slot_serializer : payload_serializer<generic_event::payload_type_t::PayloadType_Slates> {
       static generic_event::payload_buffer_t event(const char* context, unsigned int flags, const std::vector<std::vector<uint32_t>>& action_ids,
-        const std::vector<std::vector<float>>& pdfs, const std::string& model_version, const std::vector<std::string>& slot_ids) {
+        const std::vector<std::vector<float>>& pdfs, const std::string& model_version, const std::vector<std::string>& slot_ids, const std::vector<int>& baseline_actions = std::vector<int>()) {
```

Comment:
it looks like you are passing in a vector in all the method calls, does it need a default?